data/ships.txt

```diff
@@ -2264,7 +2274,8 @@ ship "Rainmaker"
 	outfits
 		"Energy Blaster" 2
 		"Meteor Missile Launcher" 4
-		"Meteor Missile" 140
+		"Meteor Missile" 284
+		"Meteor Missile Box" 8
```

Comment:
next to the Cruiser the biggest storage buff; double the missiles, double the fun
I can bring it back to 6 or 4 but I prefer to not go below 4